### DISCUSSION TITLE
Custom fail sound in skins.

### DISCUSSION THREAD

**Discussion by Hasster1** [ppy/osu#21579] [Ideas] [answered]
Custom fail sound when failing.
osu!stable has it, but i guess people have forgotten about it, since i didn't see anyone mentioning it anywhere.

**Comment by Hasster1** [comment]
No wait, other issues are about the section fail/pass, not the general fail. There is still no mention of the actual fail sound not being used.

**Reply by Schweeeeeeeeeeeeeeee** [reply]
I'm pretty sure it's already a feature, your skin just doesn't have a custom fail sound.

Edit: Just checked, it's already a feature
Works with this [skin](https://osu.ppy.sh/community/forums/topics/191909?n=1), you could check what the fail sound file is named and just add a custom fail sound yourself.

**Reply by Hasster1** [reply]
Doesn't work for me, even with this skin i still can't hear a fail sound. Also, the name of the file is the same.

**Comment by Schweeeeeeeeeeeeeeee** [comment]
You might need to turn off Beatmap hitsounds

![image](https://user-images.githubusercontent.com/88291316/206486974-5b7c881c-826e-4fcb-9973-573db4c49ccd.png)

**Reply by Hasster1** [reply]
Nope, i've disabled them all, and i still can't hear a custom fail sound from my skin.

**Comment by Hasster1** [comment, marked as answer]
Fixed.

**Reply by nagi-desuuu** [reply]
https://github.com/ppy/osu/pull/22133
